frontend/src/components/CompanyInput/CompanyInput.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import { 
+    Form, 
+    FormGroup, 
+    Input, 
+    InputGroup, 
+    InputGroupAddon, 
+    Button 
+} from "reactstrap";
+import "./CompanyInput.css";
+
+export default function CompanyInput() {
+
+    const [companyName, setCompanyName] = useState("");
```

Comment:
If were going to be using state id rather us create a class component than use the hooks in a functional component.

Reply:
What is wrong with hooks?